openmetadata-ui/src/main/resources/ui/src/utils/mocks/ApplicationUtils.mock.ts

```diff
@@ -192,211 +193,211 @@ export const MOCK_APPLICATION_ENTITY_STATS = {
   },
   {
-    name: EntityType.TEST_SUITE,
+    name: upperFirst(EntityType.TEST_SUITE),
     totalRecords: 3,
     failedRecords: 0,
     successRecords: 3,
   },
   {
-    name: EntityType.DATA_PRODUCT,
+    name: upperFirst(EntityType.DATA_PRODUCT),
     totalRecords: 0,
     successRecords: 0,
     failedRecords: 0,
   },
   {
-    name: EntityType.SEARCH_INDEX,
+    name: upperFirst(EntityType.SEARCH_INDEX),
     totalRecords: 1,
     failedRecords: 0,
     successRecords: 1,
   },
   {
-    name: EntityType.GLOSSARY_TERM,
+    name: upperFirst(EntityType.GLOSSARY_TERM),
     totalRecords: 0,
     successRecords: 0,
     failedRecords: 0,
   },
   {
-    name: EntityType.SEARCH_SERVICE,
+    name: upperFirst(EntityType.SEARCH_SERVICE),
     totalRecords: 1,
     failedRecords: 0,
     successRecords: 1,
   },
   {
-    name: EntityType.CLASSIFICATION,
+    name: upperFirst(EntityType.CLASSIFICATION),
     totalRecords: 3,
     failedRecords: 0,
     successRecords: 3,
   },
   {
-    name: EntityType.DATABASE_SCHEMA,
+    name: upperFirst(EntityType.DATABASE_SCHEMA),
     totalRecords: 4,
     failedRecords: 0,
     successRecords: 4,
   },
   {
-    name: EntityType.MLMODEL_SERVICE,
+    name: upperFirst(EntityType.MLMODEL_SERVICE),
     totalRecords: 1,
     failedRecords: 0,
     successRecords: 1,
   },
   {
-    name: EntityType.STORAGE_SERVICE,
+    name: upperFirst(EntityType.STORAGE_SERVICE),
     totalRecords: 1,
     failedRecords: 0,
     successRecords: 1,
   },
   {
-    name: EntityType.DATABASE_SERVICE,
+    name: upperFirst(EntityType.DATABASE_SERVICE),
     totalRecords: 3,
     failedRecords: 0,
     successRecords: 3,
   },
   {
-    name: EntityType.METADATA_SERVICE,
+    name: upperFirst(EntityType.METADATA_SERVICE),
     totalRecords: 1,
     failedRecords: 0,
     successRecords: 1,
   },
   {
-    name: EntityType.PIPELINE_SERVICE,
+    name: upperFirst(EntityType.PIPELINE_SERVICE),
     totalRecords: 1,
     failedRecords: 0,
     successRecords: 1,
   },
   {
-    name: EntityType.STORED_PROCEDURE,
+    name: upperFirst(EntityType.STORED_PROCEDURE),
     totalRecords: 12,
     failedRecords: 0,
     successRecords: 12,
   },
   {
-    name: EntityType.DASHBOARD_SERVICE,
+    name: upperFirst(EntityType.DASHBOARD_SERVICE),
     totalRecords: 2,
     failedRecords: 0,
     successRecords: 2,
   },
   {
-    name: EntityType.ENTITY_REPORT_DATA,
+    name: upperFirst(EntityType.ENTITY_REPORT_DATA),
     totalRecords: 4,
     failedRecords: 0,
     successRecords: 4,
   },
   {
-    name: EntityType.MESSAGING_SERVICE,
+    name: upperFirst(EntityType.MESSAGING_SERVICE),
     totalRecords: 1,
     failedRecords: 0,
     successRecords: 1,
   },
   {
-    name: EntityType.INGESTION_PIPELINE,
+    name: upperFirst(EntityType.INGESTION_PIPELINE),
     totalRecords: 4,
     failedRecords: 0,
     successRecords: 4,
   },
   {
-    name: EntityType.DASHBOARD_DATA_MODEL,
+    name: upperFirst(EntityType.DASHBOARD_DATA_MODEL),
```

Comment:
@Ashish8689, instead of doing this I think we can loop over and just update the name with upperFirst.